test/runtests.jl

```diff
@@ -6,4 +6,5 @@ using Test
     include("header.jl")
     include("io.jl")
     include("data.jl")
+    include("consistency.jl")
```

Comment:
Recent versions of MicroMamba are bounded at Julia v1.6, so on v1.3, we get an older version of CondaPkg, which fails here. Best to only test consistency for v1.6 and newer:
```suggestion
    VERSION >= v"1.6" && include("consistency.jl")
```